webapp/src/modules/publication/selectors.js

```diff
@@ -34,3 +37,18 @@ export const getPublications = createSelector(
     }
   }
 )
+
+export const getPublicationByCoordinate = (state, x, y) => {
+  const parcels = getParcels(state)
+  const parcel = parcels[buildCoordinate(x, y)]
+
+  if (parcel) {
+    const publications = getData(state)
+    const parcelPublications = findAssetPublications(
+      publications,
+      parcel,
+      PUBLICATION_STATUS.open
+    )
+    return parcelPublications[0]
+  }
```

Comment:
could we return `null` instead of `undefined` if the parcel is not in the state?

Reply:
👍 !